pkg/virt-controller/watch/migration.go

```diff
@@ -222,19 +224,23 @@ func (c *MigrationController) runWorker() {
 }
 
 func (c *MigrationController) Execute() bool {
-	key, quit := c.Queue.Get()
+	queue := &c.Queue
+	if c.Queue.Len() == 0 && c.pendingQueue.Len() > 0 {
+		queue = &c.pendingQueue
+	}
+	key, quit := (*queue).Get()
```

Comment:
Do we actually need `(*queue)` ?

Reply:
Surprisingly yes, not sure why, thought go would deref for you as usual...